cmd/csi-snapshotter/main.go

```diff
@@ -22,9 +22,13 @@ import (
 	"fmt"
 	"os"
 	"os/signal"
+	//"strconv"
```

Comment:
Can you remove this since it is not needed?

Reply:
fixed.